src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.cs

```diff
@@ -38,6 +39,29 @@ internal partial class DefiniteAssignmentPass : LocalDataFlowPass<
         DefiniteAssignmentPass.LocalState,
         DefiniteAssignmentPass.LocalFunctionState>
     {
+        /// <summary>
+        /// A mapping from local variables to the index of their slot in a flow analysis local state.
+        /// </summary>
+        private readonly PooledDictionary<VariableIdentifier, int> _variableSlot = PooledDictionary<VariableIdentifier, int>.GetInstance();
```

Comment:
>_variableSlot [](start = 67, length = 13)

It looks naming of new fields is inconsistent around the use of "_".

Reply:
These fields were moved from the base class, with the names unchanged. The names having a leading "_" for `private` fields, and no "_" for `protected` fields.

---
In reply to: [561371496](https://github.com/dotnet/roslyn/pull/50417#discussion_r561371496) [](ancestors = 561371496)